examples/main.tf

```diff
@@ -22,20 +22,22 @@ resource "prefect_service_account" "test" {
   api_keys = [
     {
       name       = "key1"
-      expiration = "2015-10-21T00:00:00+11:00"
+      expiration = "2015-10-21T00:00:00+00:00"
     },
     {
       name       = "key2"
-      expiration = "2015-10-21T00:00:00+11:00"
+      expiration = "2015-10-21T00:00:00+00:00"
     }
   ]
 }
 
-resource "aws_secretsmanager_secret" "this" {
-  name = "prefect-svc.apikey"
-}
+# example showing how to store the api key in an AWS Secrets Manager secret
 
-resource "aws_secretsmanager_secret_version" "this" {
-  secret_id     = aws_secretsmanager_secret.this.id
-  secret_string = prefect_service_account.test.api_keys[0].key
-}
+# resource "aws_secretsmanager_secret" "this" {
```

Comment:
Do you want this commented out? I don't see an issue with it uncommented as it's an example

Reply:
Yeh I had to comment this out otherwise when dev overrides are enabled `terraform plan` fails (see the error in the PR description).

I think the common practice here is to have minimal examples that only use a single terraform provider .. for more info see https://github.com/hashicorp/terraform/issues/27459#issuecomment-1382126220.